vars/buildExecute.groovy

```diff
@@ -102,7 +108,12 @@ void call(Map parameters = [:]) {
                     error "[${STEP_NAME}] buildTool not set and no dockerImage & dockerCommand provided."
                 }
         }
-
+        if (config.buildTool != 'docker' && config.cnbBuild) {
+            if (!['npm', 'maven', 'mta', 'docker'].contains(config.buildTool)) {
```

Comment:
`docker` is excluded by the condition above

Reply:
Ah I didn't notice that. The person that created this PR is on vacation- I'm guessing that when the docker build tool is selected then it doesn't make sense to run a cnbBuild, right? I'll remove docker from line 112

Reply:
I've removed docker as an option (and removed it from the top check because I don't know why it should get special treatment there)